### ISSUE TITLE
Performance affected by inserting a flight plan

### ISSUE THREAD

**Issue by mstredansky** [flybywiresim/a32nx#5121]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-14T09:02:20+00:00",
  "ref": "master",
  "sha": "4eaf1e43033d8ce8ae96027ab4449462121417fc",
  "actor": "MisterChocker",
  "event_name": "manual"
}
```

### Describe the bug

Significant drop in performance after inserting a flight plan into the MCDU (both from Simbrief and manual input). FPS remains unaffected, but the overall smoothness of rendering is affected. The issue is most visible in outside view while circling the view around the plane. The camera movement is smooth at the beginning, but at certain point the rendering freezes and then rapidly resumes.
The performance parameters as CPU and GPU utilization or FPS count are not affected after inserting a flight plan (when this behaviour beggins).

The only way to reduce this performance hit is by turning off the ND displays, but it helps only a little. Once the plane is powered down, the issue is no longer present.

The longer the flight plan, the bigger the performance drop. The performance drop is present regardless of the used graphics settings (tested also on absolutely minumum settings without any effect on the issue).

It looks like the flightplan being added affects the coherentGT draw the most. The issue affects both Stable and Developmental versions.

### Expected behavior

To be able to load a flight plan without a significant performance drop again.

### Steps to reproduce

1. Load the aircraft
2. Turn on the main systems (External power, ADIRS etc.)
3. Enter a flight plan in the MCDU and pan the camera around in both inside and outside views
4. Try turning off the ND displays and observe if it helps (in my case it sometimes does)
5. Power down the aircraft and notice the performance stabilise
6. EXTRA STEP: I have noticed a small performance drop also after turning on the flyPad

### References (optional)

4 videos demonstrating the issue uploaded to Google Drive:

https://drive.google.com/drive/folders/1PHUlQCVyCTpApepb1szP1rPYAxgtGeKH?usp=sharing

1. File: MSFS FBW PERFORMANCE_SHORTER FP - Demonstration of the issue using a shorter route and default livery
2. File: MSFS FBW PERFORMANCE_VERY LONG FP - Demonstration of the issue using a long route and default livery
3. File: MSFS FBW PERFORMANCE_DEPARTURE_SHORTER FP - Demonstration of the issue using a shorter route during taxi and departure in a custom livery
4. File: MSFS FBW PERFORMANCE_CRUISE_SHORTER FP - Demonstration of the issue using a shorter route during cruise in custom livery

The additional custom livery videos are provided to prove that a custom livery has no impact on the issue. Tested also with an empty Community folder with the same results.

### Additional info (optional)

Example of a long route with the biggest performance hit (used in the two on ground videos from References):

LZIB/RWY13 TABI1Y TABIN Q350 ERGOM DCT BUD DCT OSLEN DCT TEGRI L605 BULEN L744 UDROS M859 KARDE N644 LAGAS A910 BARAD M747 SUBUT T923 ERLEV M11 RODAR A909 BABUM A477 NONIK B361 DUTAR A845 DAVET P173 TAPIS L509 LAJAK G796 HANGU J139 NONIB L509 SULOM A466 ASARI J24 SAM J21 DPN R460 LKN G598 SMR SMRA2R RNAV02/RATAN/VNKT

Example of a shorter/mid range route with lower performance hit (used in the 2 in flight videos from References):

LZIB/RWY13 BERV1A BERVA P10 UPLAV L156 SOPAV L617 ADVAB L619 DENKO L996 VAVUN M602 BINKA SONAL KOVIK GUNPA 63N010W ING G3 NASBU NASB4N/ILS10Z/SONAX/BIKF

Hardware used:
CPU - AMD Ryzen 5 5600X
GPU - Palit Gamerock NVIDIA RTX 3070
RAM - Patriot Viper Steel Series 32 GB Dual Channel 3200 MHz
SIM DRIVE - Samsung EVO 970 1 TB M.2 NVME
SYSTEM DRIVE - ADATA XPG 8200 PRO 512 GB M.2 NVME
MONITOR - Native 1440p G-SYNC Compatible

### Discord Username (optional)

Liliputan#8440

**Comment by Benjozork** [comment]
Fixed by #4639

**Comment by Venorcis** [comment]
Just wanted to add that I experienced this too (drop to like 5 FPS also causing the auto-pilot to sway up and down), and that as a work-around, doing a DIRECT to a bit further away waypoint (clearing some others) resolved the slowdown for me at that time.
